### PR TITLE
Hello-Microverse-Project

### PR DESCRIPTION
**Pull request by AmaliaGM**
Greetings!! This is Amalia. I´ve been assigned for you to review my code. Please review my project. Thanks in advance for you´re advice.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (sja-thedude) [inline .gitignore:1]: - [ ] Kindly make sure to also add node_modules to the .gitignore file as per the requirement.

File: .github/workflows/-github-workflows-linters.yml.url [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[InternetShortcut]
+URL=https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml
```
Comment (sja-thedude) [inline .github/workflows/-github-workflows-linters.yml.url]: - [ ] Kindly delete this file and setup linters as asked in the summary.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @AmaliaGM, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct git-flow
✔️ Good job so far

## Required 
- [ ] Kindly setup your linters correctly. Delete the existing .github/workflows and Make sure to add all the files except readme from [here](https://github.com/microverseinc/linters-config/tree/master/html-css). Also, make sure all the linters pass and have a green checkmark.
- [ ] Kindly update your readme file with your contact details and delete the unnecessary heading which is not required.
- [ ] Kindly add a descriptive Pull request title as shown in the image marked 1. Replace `Working station` with `Hello Microverse Project`.
- [ ] Kindly add a pull request description as shown in the image marked 2. Make sure to add the description by clicking the 3 dots on the right side.

2022-05-25 (2)_LI

- [ ] _Check the comments under the review._

## Optional suggestions
- [ ] Kindly make sure from next time name your branch more meaningful.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaliaGM — COMMENTED** [review]
hi.. I did the changes but as soon as I deleted the headding it went red and no workflow is running (I think). Thanks for your advices. By the way I have no idea how I closed this review, sorry.. I know I wasn´t suppose to

**Comment by sja-thedude** [comment]
@AmaliaGM you have to reopen this PR , currently it’s closed!

**Comment by AmaliaGM** [comment]
I dont know how.. tried to request a review of the changes

**Comment by AmaliaGM** [comment]
@sja-thedude done! Sorry about that

**Comment by AmaliaGM** [comment]
@sja-thedude Hi. It came to my knowing  that you can not rename a branch while in a pull request (it closes by defect). So I won't be able to do it without closing the request.

**Comment by sja-thedude** [comment]
> @sja-thedude Hi. It came to my knowing  that you can not rename a branch while in a pull request (it closes by defect). So I won't be able to do it without closing the request. 

@AmaliaGM I have kept the renaming of branch as optional! You have to rename the pull request title and description!

**Comment by AmaliaGM** [comment]
Great. Thanks for your response.

File: README.md [hunk 3]
```diff
@@ -1,28 +1,8 @@
...
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-To get a local copy up and running follow these simple example steps.
+# Getting Started
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- As required by the previous reviewer, please kindly update your readme and delete the unnecessary heading.
above

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AmaliaGM 


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- As the other reviewer missed this part, please consider them a a mandatory requirement. Please update your readme by adding all the missing tags from the [microverse readme template](https://github.com/microverseinc/readme-template) to make your reamde more professional.

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaliaGM** [comment]
@zieeco thanks for your feedback will make the changes.

**Review by Munsa1 — APPROVED** [review]
Hi @AmaliaGM  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive readme
- Passing linters
- Correct gitflow
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaliaGM** [comment]
@Munsa1 Thanks for your review! Happy codding!

**Comment by AmaliaGM** [comment]
How?

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by AmaliaGM** [comment]
Van I please get some help with the linter set up? As for the workflow,if I erase that file,does not work at all.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**